libs/api-spec/api-spec.yml

```diff
@@ -2471,3 +2524,40 @@ components:
                 type: string
                 example: after meals
       description: Request body to update prescription
+    ExistingMedicalRecord:
+      content:
+        application/json:
+          schema:
+            type: object
+            x-examples:
+              example-1:
+                patient_status: string
+                symptoms: 'fever, sneeze'
+                icd10_code: A750
+                suggestions: full rest 10 days
+                patient_id: 516a962a-802f-41f5-a2e0-c315146ad2d4
+                polyclinic_id: 1
+            properties:
+              patient_status:
+                type: string
+                enum:
+                  - OUTPATIENT
+                  - REFERRED
+                example: OUTPATIENT
+              symptoms:
+                type: string
+                example: fever
+              icd10_code:
+                type: string
+                example: A750
+              suggestions:
+                type: string
+                example: bed rest
+              patient_id:
+                type: string
+                example: 516a962a-802f-41f5-a2e0-c315146ad2d4
+              polyclinic_id:
+                type: integer
+                example: 1
+          examples: {}

```

Comment:
Delete this too

Reply:
Applied https://github.com/ALTERA-CAPSTON-41/backend/pull/68/commits/dbdb94f0b1c19b7a7091901f8cb6ac6ff754bae7.